File: python/exercicio_1.py
"""
1 - Faça um algoritmo que leia os valores de A, B, C e em seguida imprima na tela a soma entre A e B é mostre se a soma é menor que C.
"""

# Lendo os valores
A = int(input("Digite o valor de A: "))
B = int(input("Digite o valor de B: "))
C = int(input("Digite o valor de C: "))

soma = A+B # somando A + B

print(f"SOMA DE A + B = {soma}") # imprimindo a soma de A + B

if soma < C: # condição if para verificar se a soma é menor que C
    print("a soma de A + B é menor que C")

File: lua/exercicio_1.lua
--[[
1 - Faça um algoritmo que leia os valores de A, B, C e em seguida imprima na tela a soma entre A e B é mostre se a soma é menor que C.
]]

-- Lendo os valores
print("Digite o calor de A")
local A = io.read("*n")

print("Digite o calor de B")
local B = io.read("*n")

print("Digite o calor de C")
local C = io.read("*n")

local soma = A + B

print("SOMA DE A + B = "..soma.."")

if soma < C then
    print("a soma de A + B é menor que C")
end